ui/utilities/visibility/flavors/responsive/index.markup.md

```diff
@@ -6,101 +6,105 @@
       <tr class="site-text-heading--label">
         <th scope="col"><span class="slds-assistive-text">Class Name</span></th>
         <th scope="col">Less than 320px</th>
-        <th scope="col">X-Small (320px)</th>
-        <th scope="col">Small (480px)</th>
-        <th scope="col">Medium (768px)</th>
-        <th scope="col">Large (1024px)</th>
-        <th scope="col">Greater than 1024px</th>
+        <th scope="col">X-Small (>= 320px)</th>
+        <th scope="col">Small (>= 480px)</th>
+        <th scope="col">Medium (>= 768px)</th>
+        <th scope="col">Large (>= 1024px)</th>
+        <th scope="col">X-Large (>= 1280px)</th>
       </tr>
       <tr>
-        <th><code>.slds-x-small-show</code></th>
-        <td class="hidden">Hide</td>
-        <td class="visible">Show</td>
-        <td class="visible">Show</td>
-        <td class="visible">Show</td>
-        <td class="visible">Show</td>
+        <th><code>.slds-hide--x-small</code></th>
         <td class="visible">Show</td>
+        <td class="hidden">Hide</td>
+        <td class="hidden">Hide</td>
+        <td class="hidden">Hide</td>
+        <td class="hidden">Hide</td>
+        <td class="hidden">Hide</td>
       </tr>
       <tr>
-        <th><code>.slds-small-show</code></th>
-        <td class="hidden">Hide</td>
+        <th><code>.slds-show--x-small</code></th>
         <td class="hidden">Hide</td>
         <td class="visible">Show</td>
         <td class="visible">Show</td>
         <td class="visible">Show</td>
         <td class="visible">Show</td>
+        <td class="visible">Show</td>
       </tr>
+
```

Comment:
Almost there. Since this is a markdown file, the whitespace between your `tr` are breaking the overview table.

![image](https://cloud.githubusercontent.com/assets/376124/24374743/ccdf33bc-12ea-11e7-8043-fc953dbf3c43.png)

Also, there is a paragraph block above the table that is out of date now. Could you update that as well?

Thanks!